.github/actions/setup-dependencies/macos/action.yml

```
@@ -14,7 +14,7 @@ runs:
       shell: bash
       run: |
         brew update
-        brew install ninja extra-cmake-modules temurin@17
+        brew install ninja extra-cmake-modules temurin@17 libarchive
```

```suggestion
        brew install ninja extra-cmake-modules temurin@17
```
vcpkg is used instead now